addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -414,7 +414,19 @@ export class HtmlField extends Component {
         const strippedPropValue = stripHistoryIds(String(this.props.record.data[this.props.name]));
         const strippedEditingValue = stripHistoryIds(this.getEditingValue());
         const domParser = new DOMParser();
-        const parsedPropValue = domParser.parseFromString(strippedPropValue || '<p><br></p>', 'text/html').body;
+        const editable = domParser.parseFromString(strippedPropValue || '<p><br></p>', 'text/html').body;
+        // Temporarilly append the editable to the DOM because the
+        // wysiwyg.getValue can indirectly call methods that needs to have
+        // access the node.ownerDocument.defaultView.getComputedStyle.
+        // By appending the editable to the dom, the node.ownerDocument will
+        // have a `defaultView`.
+        const div = document.createElement('div');
+        div.style.display = 'none';
+        div.append(editable);
+        document.body.append(div);
```

Comment:
Is it safe to append to global `document`?

Reply:
It seems safe enough to me.

Reply:
Could you elaborate?

Reply:
?
I'd like to know your reasoning before moving on with this.

Reply:
As the element is will not visible, I don't see how that can be problematic.

Reply:
I see, thanks. My concern was with things we've had before where we relied on certain documents to match each other (like with Owl using `instanceof`). If you don't think any of that will be applied to this element then I trust you, I just wanted to be sure you considered potential issues.